js/src/scrollspy.js

```diff
@@ -269,7 +269,7 @@ const ScrollSpy = (($) => {
     static _jQueryInterface(config) {
       return this.each(function () {
         let data    = $(this).data(DATA_KEY)
-        let _config = typeof config === 'object' && config || null
```

Comment:
I agree with @glebm and I prefer to work with a null value than an undefined because we use ``strict mode`` and it will throw an error